install/memsql/install_memsql_cluster.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2015 Insight Data Science
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# check input arguments
+if [ "$#" -ne 1 ]; then
+    echo "Please specify cluster name!" && exit 1
+fi
+
+PEG_ROOT=$(dirname ${BASH_SOURCE})/../..
+source ${PEG_ROOT}/util.sh
+
+CLUSTER_NAME=$1
+
+MASTER_DNS=$(fetch_cluster_master_public_dns ${CLUSTER_NAME})
+
+script="${PEG_ROOT}/install/memsql/install_memsql.sh"
+run_script_on_node ${MASTER_DNS} ${script}
```

Comment:
That's right. You then use the GUI to finish the deployment. It's quite intuitive. I give instructions after you install it on the single node